src/sentry/relay/config/metric_extraction.py

```diff
@@ -28,9 +31,12 @@
 # Version of the metric extraction config.
 _METRIC_EXTRACTION_VERSION = 1
 
-# Maximum number of custom metrics that can be extracted for alert rules with
+# Maximum number of custom metrics that can be extracted for alerts and widgets with
 # advanced filter expressions.
-_MAX_ALERT_METRICS = 100
+# TODO: remove this, or enforce limits for alerts and widgets separately.
```

Comment:
Is this left by accident or is still a TODO.
Nit: please mark TODOs that are left for a long time with your name (and maybe a date) so somebody that
comes a year for now and sees the TODO knows what is going on, example.
# TODO RaduW 3.07.2023 I need to fix this when we get feature X implemented